Runtime/Client/LootLockerServerRequest.cs

```diff
@@ -116,6 +116,15 @@ public class LootLockerErrorData
         /// If the request was not a success this property will hold any error messages
         /// </summary>
         public string message { get; set; }
+
+        /// <summary>
+        /// An easy way of debugging LootLockerErrorData class, example: Debug.Log(onComplete.errorData);
+        /// </summary>
+        /// <returns>string used to debug errors</returns>
+        public override string ToString()
+        {
+            return $"We encountered an unexpected server error. Please try again later.\n If the issue persists, please contact LootLocker support and reference the following error details:\n trace ID - {trace_id},\n request ID - {request_id},\n message - {message}.";
```

Comment:
```suggestion
            return $"An unexpected server error occured, please try again later.\n If the issue persists, please contact LootLocker support and provide the following error details:\n trace ID - {trace_id},\n request ID - {request_id},\n message - {message}.";
```

I think generally having a ToString on errorData is fantastic. Can you make sure that it prints nicely in editor though? It's rather long, and we already have feedback that the important part of the error messages disappear out of the log line in Unity because of the length.

I also kinda object to this since "Please try again later" means that it is a problem on **our** end. If it's a 4xx error then they can retry all they want, it's not gonna change anything. So we probably should at least distinguish those cases.